gen/installer/aws.py

```diff
@@ -138,6 +215,24 @@ def get_test_session(config=None):
     return release.call_matching_arguments(release.storage.aws.get_session, config, True)
 
 
+def gen_ami_mapping(mappings):
```

Comment:
To build the final dictionary could go even more one line removing the manual iteration over mappings with a dictionary comprehsion (although not necessarily a good thing):
```python
final[region] = {'default' if key == 'natami' else key: amis[key] for key in mappings}
```